package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fastify-postgres",
-  "version": "1.3.2",
+  "version": "2.0.0",
```

Comment:
Can you please revert this? We bump the version when we release.

Reply:
Should be reverted, I assume we'll squash commits before merge.